axlearn/common/compiler_options_test.py

```diff
@@ -24,10 +24,10 @@ def f(x: Tensor) -> Tensor:
         )
         self.assertEqual(f_compiled(5), 15)
 
-    def atest_xla_flags_from_options(self):
```

Comment:
this test was never being run because of the `a` before `test` in function name.